moveit_core/constraint_samplers/test/test_constraint_samplers.cpp

```diff
@@ -1105,14 +1105,14 @@ TEST_F(LoadPlanningModelsPr2, SubgroupPoseConstraintsSampler)
     if (s->sample(ks, ks_const, 1))
       succ++;
   }
-  ROS_INFO_NAMED("pr2_arm_kinematics_plugin",
-                 "Success rate for IK Constraint Sampler with position & orientation constraints for both arms: %lf",
-                 (double)succ / (double)NT);
+  // ROS_INFO_NAMED("pr2_arm_kinematics_plugin",
+  //                "Success rate for IK Constraint Sampler with position & orientation constraints for both arms: %lf",
+  //                (double)succ / (double)NT);
 }
 
 int main(int argc, char** argv)
 {
   testing::InitGoogleTest(&argc, argv);
-  ros::Time::init();
+  // ros::Time::init();
```

Comment:
remove dead code don't just comment it out

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/abf4a74859bb5f3ca50a946b9bd0f48f1db59fa5